common/_variables.scss

```
@@ -3,6 +3,10 @@ $main-color: #ff8000 !default;
 $link-color: #126f9a !default;
 $header-bg: #fff !default;
 $footer-bg: #fff !default;
+$header-btn-bg: $header-bg !default;
+$header-btn-bottom-border-width: 2px !default;
```

To make it similar to the legacy pages

```suggestion
$header-btn-bottom-border-width: 4px !default;
```